From bcbcf94f3224c2cf5c312b41667187c6b2e6d3ee Mon Sep 17 00:00:00 2001
From: Sergey Larin <cerg2010cerg2010@mail.ru>
Date: Thu, 20 Jun 2019 20:12:18 +0300
Subject: [PATCH] ARM: dts: tegra20-glide: Fix STMPE GPIO binding

These GPIOs are also used as interrupts.

norequest-mask is set to avoid conflicts with keyboard.

Signed-off-by: Sergey Larin <cerg2010cerg2010@mail.ru>
---
 arch/arm/boot/dts/tegra20-glide.dts | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/arch/arm/boot/dts/tegra20-glide.dts b/arch/arm/boot/dts/tegra20-glide.dts
index e9a05dccb734..90f2a6eb9d77 100644
--- a/arch/arm/boot/dts/tegra20-glide.dts
+++ b/arch/arm/boot/dts/tegra20-glide.dts
@@ -1173,8 +1173,11 @@
 
 			stmpegpio: stmpe_gpio {
 				compatible = "st,stmpe-gpio";
+				st,norequest-mask = <0x7fff>;
 				gpio-controller;
 				#gpio-cells = <2>;
+				interrupt-controller;
+				#interrupt-cells = <2>;
 			};
 		};
 	};
-- 
2.22.0

